frontend/src/components/browse/BrowseContent.tsx

```diff
@@ -103,12 +102,9 @@ export default function BrowseContent({
 
   const token = new Cookies().get("auth_token");
 
-  const {
-    filters,
-    handleUpdateFilterValues,
-    handleSetErrorMessage,
-    handleApplyNewFilters: applyNewFilters,
-  } = useContext(FilterContext);
```

Comment:
Same here, `handleUpdateFilterValues` is provided by FilterContext and not used in BrowseConent. It was only used to propagate down to components. 
Therefore, it can be removed and looked up on demand.